appservice/src/deploy/localGitDeploy.ts

```diff
@@ -26,6 +26,7 @@ export async function localGitDeploy(client: SiteClient, fsPath: string): Promis
     const password: string = nonNullProp(publishCredentials, 'publishingPassword');
     const remote: string = `https://${username}:${password}@${client.gitUrl}`;
     const localGit: git.SimpleGit = git(fsPath);
+    await verifyNoRunFromPackageSetting(client);
```

Comment:
Can we move this after we check for uncommitted changes? Feels unnecessary to update the application settings unless we know they're actually going to deploy